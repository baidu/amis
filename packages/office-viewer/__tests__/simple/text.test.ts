import {snapShotTest} from '../snapShotTest';

test('text', async () => {
  snapShotTest('./docx/simple/text.xml');
});
