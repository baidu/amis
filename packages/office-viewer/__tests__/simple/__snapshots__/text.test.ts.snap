// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`text 1`] = `
<div
  class="docx-viewer-0 docx-viewer-wrapper"
  id="root"
>
  <style>
    
  
  .docx-viewer-wrapper {

  }

  .docx-viewer-wrapper &gt; article &gt; section {
    background: white;
  }

  /** docDefaults **/

  .docx-viewer-0 p {
    margin: 0;
    padding: 0;
    line-height: 1.5;
  }

  .docx-viewer-0 table {
    border-spacing: 0;
  }

  .docx-viewer-0 .docx-viewer-0-p {
    
  }

  .docx-viewer-0 .docx-viewer-0-r {
    white-space: pre-wrap;
    overflow-wrap: break-word;
    font-family: var(--docx-theme-font-minorHAnsi), var(--docx-theme-font-minorEastAsia);
font-size: 16.00px;

  }
  

  
    
      .docx-viewer-0 .docx-viewer-0-Normal {
        text-align: justify;

      }
      
    
      .docx-viewer-0 .docx-viewer-0-Normal &gt; .docx-viewer-0-r {
        
      }
      
    
    
    
    
    
    
    
    
    
    
    
 .docx-viewer-0 .docx-viewer-0-TableNormal {
  border-collapse: collapse;
  margin-left: 0.00px;

 }

 .docx-viewer-0 .docx-viewer-0-TableNormal &gt; tbody &gt; tr &gt; td {
  padding-top: 0.00px;
padding-left: 7.20px;
padding-bottom: 0.00px;
padding-right: 7.20px;

 }
 
    
    
    
    
    
    
    
  
  </style>
  <style>
    /** embedded fonts **/
  </style>
  <article>
    <section
      style="position: relative; width: 815.98px; padding: 96.00px 96.00px 96.00px 96.00px;"
    >
      <p
        class="docx-viewer-0-p"
      >
        <span
          class="docx-viewer-0-r"
        >
          <div
            style="position: absolute; left: 0.00px; top: -0.33px; width: 597.19px; height: 548.08px; background-color: rgb(23, 41, 72);"
          >
            <p
              class="docx-viewer-0-p"
              style="text-align: center;"
            >
              <span
                class="docx-viewer-0-r"
                style="font-family: "思源宋体 CN"; color: rgb(47, 84, 150); font-size: 173.33px;"
              >
                合格证
              </span>
            </p>
          </div>
        </span>
      </p>
    </section>
    <section
      style="position: relative;"
    />
  </article>
</div>
`;
